docs/source/logging/logging.md

```diff
@@ -19,19 +23,50 @@ log.info("Send information")
 ```
 
 ```{note}
-The name of a logger corresponds to a key in the `loggers`  section in `logging.yml` (e.g. `kedro.io`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+The name of a logger corresponds to a key in the `loggers`  section in `logging.yml` (e.g. `kedro`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+```
+
+You can take advantage of rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) when enabled in your logging calls: 
```

Comment:
I'm wondering if it makes more sense to swap this section and the next, since we explain/link to Rich below, but already mention it here.

Reply:
Yeah, I was wondering that. My logic for this was that "Use logging" is more useful to most people than the following sections, but actually looking at it again I'm not sure that's true. Let me try to rejig it a bit.